### ISSUE TITLE
OAS-specificatie voor het opvragen van STATUSsen is niet duidelijk

### ISSUE THREAD

**Issue by HenriKorver** [VNG-Realisatie/gemma-zaken#1469]
Het is niet duidelijk dat je in de query-parameter `zaak` *meerdere* url's mag meegeven (zie screen shot onderaan), hoewel dit cruciaal is als je op efficiente wijze de statusdetails van meerdere zaken tegelijk wilt ophalen, zie ook https://github.com/Geonovum/KP-APIs/issues/173.

![afbeelding](https://user-images.githubusercontent.com/37145898/69256540-e65e0c80-0bb9-11ea-98f0-55c9244f84e2.png)

**Comment by stevenbal** [comment]
Ik heb even getest of het mogelijk is om te filteren op meerdere `zaak` URLs, maar dit is in de referentieimplementatie nog niet mogelijk, dus ik heb hiervoor een aparte user story aangemaakt: https://github.com/VNG-Realisatie/gemma-zaken/issues/1590

**Comment by joeribekker** [comment]
De vraag is nu, moeten we dit toevoegen of niet? @Hugo-ter-Doest @HenriKorver

**Comment by HenriKorver** [comment]
> Ik heb even getest of het mogelijk is om te filteren op meerdere zaak URLs, maar dit is in de referentieimplementatie nog niet mogelijk, dus ik heb hiervoor een aparte user story aangemaakt: #1590

@stevenbal Is het in de productie-omgeving wel mogelijk?

**Comment by HenriKorver** [comment]
> De vraag is nu, moeten we dit toevoegen of niet? 

Lijkt me wel. Deze feature is op het forum van het Kennisplatform API's door @sergei-maertens aangedragen als schoolvoorbeeld (zie  Geonovum/KP-APIs#173) en nu blijkt het niet te werken of zelfs niet te bestaan.

**Comment by sergei-maertens** [comment]
> @stevenbal Is het in de productie-omgeving wel mogelijk?

nee, maar dat is toch ook niet relevant? De referentieimplementatie toont het "correcte" gedrag die productie-implementaties moeten volgen.

**Comment by HenriKorver** [comment]
> nee, maar dat is toch ook niet relevant? De referentie-implementatie toont het "correcte" gedrag die productie-implementaties moeten volgen.

Dat zou inderdaad zo moeten zijn. Maar op het ogenblik slaagt de productie-implementatie wel voor de tests op api-test.nl en de referentie-implementatie-niet. Daaruit maak ik op dat men op productie voortvarender bezig is, vandaar mijn nieuwsgierigheid of het daar wel zou werken...

**Comment by HenriKorver** [comment]
[![afbeelding](https://user-images.githubusercontent.com/37145898/78884571-6e568d00-7a5b-11ea-942f-324b04b07bc5.png)](https://api-test.nl)

**Comment by michielverhoef** [comment]
Deze mogelijkheid is opgevuld door de UUID_In query parameter in de operatie zaak_zoek in Zaken API 1.3:

https://zaken-api.test.vng.cloud/api/v1/schema/#tag/zaken/operation/zaak__zoek
